src/Enterspeed.Source.UmbracoCms/Controllers/Api/DashboardApiController.cs

```diff
@@ -100,6 +100,35 @@ public IActionResult Seed()
             }
         }
 
+        [HttpPost]
+        public IActionResult CustomSeed(CustomSeed customSeed)
+        {
+            var publishConfigured = _enterspeedConfigurationService.IsPublishConfigured();
+            var previewConfigured = _enterspeedConfigurationService.IsPreviewConfigured();
+
+            if (!publishConfigured && !previewConfigured)
+            {
+                return BadRequest(
+                    new Response
+                    {
+                        Status = HttpStatusCode.BadRequest,
+                        Success = false,
+                        Message = "Enterspeed has not yet been configured"
```

Comment:
We are using this check quite a few places. Should we consider add some functionality utilizing attributes on the controller, combined with some middleware? - It will clean up the controllers a lot.